dataplane/src/main.rs

```diff
@@ -1,28 +1,28 @@
-// SPDX-License-Identifier: Apache-2.0
-// Copyright Open Network Fabric Authors
+// // SPDX-License-Identifier: Apache-2.0
+// // Copyright Open Network Fabric Authors
+//
```

Comment:
Double commenting looks like a mistake.

Reply:
this should be resolved by dropping the introducing commit